src/vm/syncblk.cpp

```diff
@@ -2554,6 +2554,17 @@ inline void LogContention()
 #define LogContention()
 #endif
 
+LONGLONG ComputeElapsedTimeInNanosecond(LARGE_INTEGER startTime, LARGE_INTEGER endTime)
```

Comment:
Maybe startTicks / endTicks instead of startTime / endTime?

Reply:
done